requirements.txt

```diff
@@ -12,6 +12,7 @@ flask==1.1.2
 graphene==3.2.1
 itsdangerous==1.1.0  # from flask
 jinja2==2.11.3  # from flask
+jsonpatch==1.21
```

Comment:
When adding a dependency to the `requirements.txt` file you also need to add it to the `debian/control` file which is used by the debian packaging.
You can look at wazo-auth `debian/control` as an example

Reply:
Thank you, It is done.